main/GarnetServer/GarnetServer.csproj

```diff
@@ -3,6 +3,29 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <ServerGarbageCollection>true</ServerGarbageCollection>
+
+    <!-- IMPORTANT: Keep the version in sync with .azure\pipelines\azure-pipelines-external-release.yml line ~6. -->
+    <Version>1.0.36</Version>
+    <PackageId>garnet-server</PackageId>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>garnet-server</ToolCommandName>
+    <Title>The Microsoft.Garnet RESP server, packaged as a .NET tool</Title>
+    <GeneratePackageOnBuild Condition="'$(Configuration)'=='Release'">True</GeneratePackageOnBuild>
+    <Authors>Microsoft</Authors>
+    <Company>Microsoft</Company>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/microsoft/garnet.git</RepositoryUrl>
+    <PackageProjectUrl>https://microsoft.github.io/garnet</PackageProjectUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
+    <Description>
+      Garnet is a remote cache-store from Microsoft Research, that offers strong performance (throughput and latency),
+      scalability, storage, recovery, cluster sharding, key migration, and replication features. Garnet uses the Redis RESP wire
+      protocol and can work with existing Redis clients.
+    </Description>
+    <PackageReleaseNotes>See https://github.com/microsoft/garnet for details.</PackageReleaseNotes>
```

Comment:
I think we should link to `https://github.com/microsoft/garnet/releases`

Reply:
This is consistent with the current nuspec so ok to leave as it is. Users can always click through to the releases.